source/_lovelace/map.markdown

```diff
@@ -18,7 +18,11 @@ type:
   type: string
 entities:
   required: true
-  description: List of entity IDs. Either this or the `geo_location_sources` configuration option is required.
+  description: List of entity IDs. Either this or `history_entities` or `geo_location_sources` configuration option is required.
+  type: list
+history_entities:
+  required: true
+  description: List of `sensor.<device-name>_geocoded_location` entities. This displays not additional to a marker at the current position of the entity a path of the location history.
```

Comment:
Can you please rephrase "This displays not additional to a marker at the current position of the entity a path of the location history."?

I don't get it. Sorry

Reply:
Sorry, musst be a mistake when I rewrote the description.  Is this better:

'It displays a path of the location history and the current-location marker if a `device_tracker` entity from the same device exists.'